app/api/v1/endpoints/prediction.py

```
@@ -0,0 +1,48 @@
+from fastapi import APIRouter, HTTPException
+from pydantic import BaseModel
```

Ruff (0.8.2)

`pydantic.BaseModel` imported but unused

Remove unused import: `pydantic.BaseModel`

(F401)